src/main/java/org/janelia/saalfeldlab/n5/N5FSReader.java

```diff
@@ -136,7 +136,7 @@ private void close() throws IOException {
 	}
 
 	/**
-	 * Reads or creates the attributes map of a group or dataset.
```

Comment:
Makes sense. I've pushed another commit that changes the behavior of `getAttributes()` in this case.